packages/@glimmer/compiler-delegates/src/module-unification/compiler-delegate.ts

```diff
@@ -73,7 +73,7 @@ export default class MUCompilerDelegate implements AppCompilerDelegate<TemplateM
     let relativePath = module.replace(/^\.\//, '');
 
     let meta;
-    if (this._builtins[name]) {
+    if ((this as any)._builtins[name]) {
```

Comment:
The type of `this` should be correctly inferred because it's a class method. Do you know why you need this cast?